Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -50,6 +51,8 @@ private function resolveInterpreterExceptionMessage(InterpreterException $except
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof InvalidRowException) {
             return $this->buildInvalidRowExceptionMessage($exception);
+        } elseif ($exception instanceof BooleanFormatException) {
```

Comment:
ok. This means that we have to have a new exception for each new interpreter etc. Can we not make a generic exception to save maintenance and overhead? or do we need to know specifically what exception it is somewhere?